### ISSUE TITLE
setTime() does not work if timepicker has been rendered already

### ISSUE THREAD

**Issue by tkrotoff** [jonthornton/jquery-timepicker#139]
Hi,

setTime() does not work if timepicker has been rendered already.
You can see it live using your [demo](http://jonthornton.github.io/jquery-timepicker/):
- Go to the "Set Time Example"
- Select a time from the timepicker (different than the current time).
- Click on "Set current time" button
- Open again the timepicker => it still shows the previous selected value (in blue) instead of the current time.

In other words, setTime() only works when called before you render the timepicker for the first time.

Edit: playing with the "Datepair Example" shows also the problem.
If you select a start-time after end-time, end-time value gets changed, not its timepicker.
